### DISCUSSION TITLE
is there a limit to where helis can crash? and is there any way to locate them besides just trying to find them in the forest or somewhere?

### DISCUSSION THREAD

**Discussion by Gamingman1** [TEHE-Studios/ExpandedHelicopterEvents#189] [Q&A] [answered]
because it seems like it did crash in the forest, but it seems like i will have to go and just look in that ENTIRE cell just to find some mediocre loot considering that ill have to do that

**Comment by Gamingman1** [comment]
yeah i found it but still the question stands

**Comment by sharkster91** [comment, marked as answer]
The only limit to where helicopters can crash is on the water or directly onto buildings.

You'll have to wait for B42 for map integration with their event system so
we can accurately mark where they crash.
